$NetBSD: patch-ab,v 1.3 2012/12/26 16:58:27 darcy Exp $

- Fix build bug with cc -g, patch submitted by john heasley.

--- gsl/gsl_math.h.orig	2004-06-10 14:17:35.000000000 -0700
+++ gsl/gsl_math.h	2004-06-28 12:44:15.000000000 -0700
@@ -129,44 +129,44 @@ double gsl_min (double a, double b);
 /* inline-friendly strongly typed versions */
 #if HAVE_EXTERN_INLINE
 
-extern inline int GSL_MAX_INT (int a, int b);
-extern inline int GSL_MIN_INT (int a, int b);
-extern inline double GSL_MAX_DBL (double a, double b);
-extern inline double GSL_MIN_DBL (double a, double b);
-extern inline long double GSL_MAX_LDBL (long double a, long double b);
-extern inline long double GSL_MIN_LDBL (long double a, long double b);
+static inline int GSL_MAX_INT (int a, int b);
+static inline int GSL_MIN_INT (int a, int b);
+static inline double GSL_MAX_DBL (double a, double b);
+static inline double GSL_MIN_DBL (double a, double b);
+static inline long double GSL_MAX_LDBL (long double a, long double b);
+static inline long double GSL_MIN_LDBL (long double a, long double b);
 
-extern inline int
+static inline int
 GSL_MAX_INT (int a, int b)
 {
   return GSL_MAX (a, b);
 }
 
-extern inline int
+static inline int
 GSL_MIN_INT (int a, int b)
 {
   return GSL_MIN (a, b);
 }
 
-extern inline double
+static inline double
 GSL_MAX_DBL (double a, double b)
 {
   return GSL_MAX (a, b);
 }
 
-extern inline double
+static inline double
 GSL_MIN_DBL (double a, double b)
 {
   return GSL_MIN (a, b);
 }
 
-extern inline long double
+static inline long double
 GSL_MAX_LDBL (long double a, long double b)
 {
   return GSL_MAX (a, b);
 }
 
-extern inline long double
+static inline long double
 GSL_MIN_LDBL (long double a, long double b)
 {
   return GSL_MIN (a, b);
